tests/core/test_impersonation_tests.py

```diff
@@ -161,14 +144,15 @@ def test_no_impersonation(self):
             "test_superuser",
         )
 
-    @unittest.mock.patch.dict("os.environ", AIRFLOW__CORE__DEFAULT_IMPERSONATION=TEST_USER)
+    @mock.patch.dict("os.environ", AIRFLOW__CORE__DEFAULT_IMPERSONATION=TEST_USER)
     def test_default_impersonation(self):
         """
         If default_impersonation=TEST_USER, tests that the job defaults
         to running as TEST_USER for a test without run_as_user set
         """
         self.run_backfill("test_default_impersonation", "test_deelevated_user")
 
+    @pytest.mark.execution_timeout(150)
```

Comment:
`TestImpersonation::test_impersonation_subdag` test almost reach timeout (60s) in the CI

```console
============================ slowest 100 durations =============================
  48.91s call     tests/core/test_impersonation_tests.py::TestImpersonation::test_impersonation_subdag
  24.39s call     tests/core/test_impersonation_tests.py::TestImpersonationWithCustomPythonPath::test_impersonation_custom
  12.27s call     tests/core/test_impersonation_tests.py::TestImpersonation::test_impersonation
  12.18s call     tests/core/test_impersonation_tests.py::TestImpersonation::test_default_impersonation
  6.40s setup    tests/core/test_impersonation_tests.py::TestImpersonation::test_impersonation
  6.34s call     tests/core/test_impersonation_tests.py::TestImpersonation::test_no_impersonation
```